src/lang/modifyAst.ts

```diff
@@ -811,40 +811,53 @@ export function addOffsetPlane({
  */
 export function addHelix({
   node,
+  axis,
+  cylinder,
   revolutions,
   angleStart,
-  ccw,
   radius,
-  axis,
   length,
+  ccw,
   insertIndex,
   variableName,
 }: {
   node: Node<Program>
+  axis?: Node<Literal> | Node<Name | CallExpression | CallExpressionKw>
+  cylinder?: VariableDeclarator
```

Comment:
Is there a reason this isn't `Node<VariableDeclarator>`? I'm fuzzy on those types.

Reply:
You're right this is what it should be based on the `extrudeNode.node.declaration` assigment. I guess it doesn't complain because Node is only about adding extra properties that I didn't need here?